Gopkg.toml

```diff
@@ -44,7 +44,11 @@
 
 [[constraint]]
   name = "github.com/deislabs/oras"
-  version = "0.4.0"
+  version = "0.5.0"
+
+[[constraint]]
```

Comment:
The latest version is `1.4.1`. Also, how about using `klog` instead given that's the de-facto standard for logging?

Reply:
unfortunately this has to do with dependency hell of the supporting libs (oras+containerd)

If you want to help clean this up, I think we should open up tickets on ORAS for support for something like `orascontext.SetLevel(oraslog.DebugLevel)` so that we do not need to import logrus here at all
https://github.com/deislabs/oras

Reply:
on the version aspect. Just trying to keep too many version mappings. Since this is all accrues to helm 3, it seemed to align better. Here’s the discussion:
https://cloud-native.slack.com/archives/CJ1KHJM5Z/p1557442141035000

Reply:
on the version aspect. Just trying to keep too many version mappings. Since this is all accrues to helm 3, it seemed to align better. Here’s the discussion:
https://cloud-native.slack.com/archives/CJ1KHJM5Z/p1557442141035000

Reply:
guessing this was meant for the other thread above